### ISSUE TITLE
BulletList formatted incorrectly on insertContent

### ISSUE THREAD

**Issue by aaravin** [aguingand/tiptap-markdown#48]
**Describe the bug**
When calling insertContent with bullet point markdown (for both numbering and bullet points), an extra bullet point/number is added at the beginning of the list.

**To Reproduce**
Pass the following input string into editor.commands.insertContent:
```
Title: The Importance of A/B Testing for SEO Success: Boosting Your Brand's Ranking on Google

Introduction:
- Briefly introduce the concept of A/B testing and its significance in the world of SEO.
- Highlight the potential impact of A/B testing on improving a brand's visibility and ranking on Google.

1. Understanding A/B Testing in SEO:
- Explain what A/B testing entails in the context of SEO.
- Discuss its purpose, which is to compare and contrast different versions of website elements to identify the one that performs better in terms of SEO metrics.
- Highlight the two variants involved in A/B testing: the control (original) version and the variant (newly implemented) version.
```
**Expected behavior**
Extra bullet/number should not appear before each list. Should work the same way as it does with setContent

**Screenshots**
<img width="709" alt="Screenshot 2023-11-05 at 11 03 09 PM" src="https://github.com/aguingand/tiptap-markdown/assets/11164041/d36d2c54-e40a-4871-b111-a1cdc018de99">

<img width="746" alt="Screenshot 2023-11-05 at 11 03 30 PM" src="https://github.com/aguingand/tiptap-markdown/assets/11164041/2a3297ae-afd8-4f9a-bd30-9ebc8c5a39f9">

**Desktop (please complete the following information):**
 - Browser: Chrome
 - Version: 0.8.3

**Comment by gediminastub** [comment]
I'm having the same issue

**Comment by aguingand** [comment]
Fixed in v0.8.7
